scripts/csv-to-json/csv/guildMottos.csv

```diff
@@ -0,0 +1,11 @@
+guildMottos
+"Sixteen of One, None of the Other"
+Until our Ancient Lord Awakes
+"Where We go, They Follow"
+"Consume, Subsume"
+"Birth, Death, Rebirth"
+Bring Back the Future
+"Truth, Discovery, and Laughter"

```

Comment:
Why are some quoted and some not?

Reply:
Because of the C in CSV, I think: if there are commas in the string, it needs quotes or else whoever is reading the CSV will get the wrong data.

Reply:
aaah yeah that looks like it